controllers/control_plane_init_locker_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	clusterv2 "sigs.k8s.io/cluster-api/api/v1alpha2"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
Package sigs.k8s.io/controller-runtime/pkg/runtime/log is deprecated
Ref: https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/runtime/log/log.go#L20